### ISSUE TITLE
Wrong version in ansible-operator with "run" command

### ISSUE THREAD

**Issue by ulrichSchreiner** [operator-framework/operator-sdk#5006]
hi,

i've installed the 1.9.0 version of `ansible-operator` as you can see with the `version` command it shows `v1.9.0`:
~~~
# ansible-operator version
ansible-operator version: "v1.9.0", commit: "205e0a0c2df0715d133fbe2741db382c9c75a341", kubernetes version: "1.20.2", go version: "go1.16.5", GOOS: "linux", GOARCH: "amd64"
~~~

But when i use the `run` command, i get this output:
~~~
# ansible-operator run
{"level":"info","ts":1624431850.9744582,"logger":"cmd","msg":"Version","Go Version":"go1.16.5","GOOS":"linux","GOARCH":"amd64","ansible-operator":"v1.8.0+git","commit":"205e0a0c2df0715d133fbe2741db382c9c75a341"}
~~~

but here the operator version is `v1.8.0+git`. this is confusing and one might ghink there are different versions of the operator installed.

**Comment by jmrodri** [comment]
@ulrichSchreiner did you do an update of your repo? I agree it is confusing, today I hit a weird issue where I did a `git fetch upstream` and for some reason did not see the v1.9.0 tag. This caused my ansible-operator to be v1.8.0+git. But now things are fixed after doing a `git fetch upstream --tags`. 

I think there is still a bug here because the version value used in run is a different field than the one used by `version`. I think we should make them consistent so that it always reports the same one regardless.

**Comment by ulrichSchreiner** [comment]
hi, 
i'm not using the source. i downlooaded the latest release with 
~~~
curl -sL https://api.github.com/repos/operator-framework/operator-sdk/releases/latest | jq -r ".assets[] | select(.name | contains(\"ansible-operator_linux_amd64\")) | .browser_download_url"
~~~
which reports v1.9.0 as version but v.1.8.0-git when the `run` command is used.

**Comment by jmrodri** [comment]
@ulrichSchreiner AH okay that's definitely bad. We have someone looking into it. Thanks for the update.

**Comment by sujil02** [comment]
@jmrodri As this PR is merged #5023 should we close it or I think we can wait for the new release to cut so we can retest this issue and then close it?
